.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "lmnopEnv\\Scripts\\python.exe"
+}
```

Comment:
Having this file in the repository is going to cause a lot of issues. Please remove it from your local repo and push your changes.
This command removes the file from your local repo.
`git rm -r --cached .vscode`